Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN apt-get update && \
   apt-get install -y --no-install-recommends \
     build-essential \
     gcc \
+    libgit2-dev \
```

Comment:
This is a great discovery! What happened to the previous build strategy without this dependency though? Is `devtools` got updated?

Reply:
Oh never mind, I just read your comment.